### PR TITLE
DSDEGP-1290 Added stack driver logging configuration

### PR DESCRIPTION
**Pull request by jacarey**
This PR adds the necessary configuration to get clio server to log to stack driver.  I'm not sure the best way to test this aside from deploying to dev server, but I'm open to other suggestions.

DSDEGP-1290
----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: Ensure that you have added or updated tests
- [x] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

**Comment by broadinst** [comment]
Can one of the admins verify this patch?

**Comment by danxmoran** [comment]
test this please

**Comment by pshapiro4broad** [comment]
add to whitelist

File: clio-server/config/docker-compose.yml.ctmpl [hunk 1]
```diff
@@ -13,6 +17,22 @@ app:
...
+  depends_on:
+    - logger
+  volumes_from:
+    - logger
+
+logger:
+  image: broadinstitute/fluentd-gcp:latest
+  expose:
+    - 5140/tcp
+    - 5140/udp
```
Comment (danxmoran) [inline clio-server/config/docker-compose.yml.ctmpl]: Do we need to expose these? They aren't listed in the [cromwell docker-compose config](https://github.com/broadinstitute/gotc/blob/develop/config/cromwell/docker-compose.yml.ctmpl#L56).

**Review by danxmoran — APPROVED** [review]
👍  looks good compared to the cromwell config, just one question

**Review by pshapiro4broad — APPROVED** [review]

File: clio-server/config/docker-compose.yml.ctmpl (hunk 1 above)
Comment (jacarey) [reply clio-server/config/docker-compose.yml.ctmpl]: Good question. I pulled this out of https://github.com/broadinstitute/gotc/blob/develop/config/elasticsearch/docker-compose.yml#L34

Maybe @hjfbynara can answer this?

Comment (jacarey) [reply clio-server/config/docker-compose.yml.ctmpl]: I'll remove them for now.. and if things don't work we can revisit. No point in exposing ports that don't need to be exposed.

**Pull request merged**